src/components/List/style.scss

```diff
@@ -0,0 +1,26 @@
+.list {
+  &__table {
+    &__data {
+      background-color: #fff;
+      border-left: 1px solid black;
+      border-right: 1px solid black;
+      text-align: center;
+      min-height: 80px;
+      padding: 20px;
+      max-width: 500px;
+      word-wrap: break-word;
+      word-break: normal;
+    }
+  }
+
+  table {
```

Comment:
Are these global styles? If so, then put them in a common file with styles. If not, then use classes